src/backend/web/app.js

```diff
@@ -1,14 +1,17 @@
 const path = require('path');
 const express = require('express');
 const expressHandlebars = require('express-handlebars');
+const Redis = require('ioredis');
```

Comment:
Instead of using ioredis directly, use our `lib/redis` and then you'll get the mock Redis stuff for free.

Reply:
@humphd thanks!